mozdownload/scraper.py

```diff
@@ -555,6 +636,39 @@ def path_regex(self):
             raise errors.NotFoundError("Specified sub folder cannot be found",
                                        folder)
 
+    """ nightly """
+    def get_checksum_url(self):
+        """Get the checksum url for nightly build."""
+        filename = os.path.split(self.url)[1]
+        filename = re.sub('\.%s$' % DEFAULT_FILE_EXTENSIONS[self.platform], '', filename)
+        filename = re.sub('\.installer$', '', filename)
+        url = urljoin(self.path, filename + '.checksums')
+        return url
+
+    def get_checksum_name(self):
+        """Get the name of the platform in the checksum file."""
+        name = ''
+        if self.platform == 'win64' or self.platform == 'win32':
+            name = 'install/sea/'
+        name = name + self.binary
+        return name
```

Comment:
Similar to what I mentioned before for re-using existing code. Basically it means that we have to replace whatever comes after the platform with the `.checksums` suffix.